### PR TITLE
Security center

### PR DESCRIPTION
**Pull request by TheLeoB**

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Reset2FA/ThirdStep/index.js [hunk 1]
```diff
@@ -14,7 +14,6 @@ class ThirdStepContainer extends React.PureComponent {
 
   onSubmit (event) {
     event.preventDefault()
-    this.setState({ timestamp: new Date().getTime(), busy: true })
```
Comment (TheLeoB) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Reset2FA/ThirdStep/index.js]: @plondon please test. thanks

File: packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/GoogleAuth/template.success.js [hunk 2]
```diff
@@ -22,6 +23,9 @@ const QRCode = styled.div`
   width: 100%;
   padding: 30px 0;
 `
+const QRCodeCopy = styled.div`
+  display: block;
+`
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/GoogleAuth/template.success.js]: `div`'s are already display: block, is this necessary?

File: packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/Yubikey/template.success.js [hunk 3]
```diff
@@ -19,6 +20,9 @@ const YubikeyContainer = styled.div`
   justify-content: center;
   align-items: center;
 `
+const YubikeyCopy = styled.div`
+  display: block;
+`
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/Yubikey/template.success.js]: Maybe use `Fragment` instead if all you need is `display: block`

**Comment by plondon** [comment]
For future branches please prefix with `feature/`

**Pull request merged**